apps/web/src/app/(pages)/(dashboard)/(roles)/page.tsx

```diff
@@ -27,8 +40,37 @@ export default function Roles() {
   return (
     <>
       {roles.isSuccess && roles.data.length > 0 && (
-        <div className="flex h-[86dvh] w-full gap-4 divide-x divide-[#474747] pl-4 lg:h-[92dvh]">
+        <div className="flex h-[86dvh] w-full gap-4 divide-x divide-[#474747] pl-4 lg:h-[92dvh] ">
           <div className="w-[28%] gap-3 overflow-auto p-1">
+            <div className="text-right">
+              <DropdownMenu>
+                <DropdownMenuTrigger className="pb-2">
+                  Filter By <ChevronDown className="inline" />
```

Comment:
Could you edit this to be similar to the design where it shows the current filter status? And then if there is no filter status we could have it say 'Sort by default' or something (or sort by None? idk question for the culture)

<img width="207" alt="Screenshot 2025-03-26 at 11 42 32 PM" src="https://github.com/user-attachments/assets/18fa20b5-3d04-47c7-97fc-5438d1a70771" />